src/deleteAll.test.js

```diff
@@ -0,0 +1,17 @@
+const clearCompleted = require('./deleteAll.js');
+
+const oldList = [
+  { description: 'Task1', completed: true, index: 1 },
+  { description: 'Task2', completed: false, index: 2 },
+  { description: 'Task3', completed: true, index: 3 },
+];
+
+const remainingList = [
+  { description: 'Task2', completed: false, index: 1 },
+];
+
+test('Delete All Completed ToDo items', () => {
+  expect(
+    clearCompleted(oldList),
+  ).toEqual(remainingList);
+});
```

Comment:
- [ ] Please kindly group your test using the describe() method. **Optionally**, you can add one or more tests in order to bring out the essence of the describe() method. A describe block is used for organizing test cases in logical groups of tests. ℹ️